Marketplace/gestureRose/gestureRose.js

```diff
@@ -0,0 +1,102 @@
+var X_TRIGGER_LIMIT = 0.1;
+var Y_TRIGGER_DIST = -0.145;
+var Z_TRIGGER_LIMIT = 0.1;
+var GLOBAL_UP = {x: 0, y: 1, z: 0};
+var DEG_TO_RAD = Math.PI / 180;
+var ANGLE_MATCH = Math.cos(45 * DEG_TO_RAD);
+
+var ROSE_SCALE = {z: 0.1297, y: 0.3165, x: 0.1241};
+var ROSE_REZ_OFFSET = {x: 0, y: 0.25, z: 0};
+var ROSE_LIFETIME = 300;
+var ROSE_DEFAULT_GRAVITY = {x: 0, y: -7, z: 0};
+var ROSE_URL = "http://fluffy.ws/HighFid/longStemRose.fbx"; //Script.resolvePath('longStemRose.fbx');
+
+var gestureStarted = false;
+var gestureStartPos = null;
+var _gestureHand = 0;
+
+Script.scriptEnding.connect(function () {
+    shutdown();
+});
+
+controllerMappingName = 'gestureRoseMapping';
+controllerMapping = Controller.newMapping(controllerMappingName);
+controllerMapping.from(Controller.Standard.LeftGrip).to(function (value) {
+    gestureLogic(value,-1);
+});
+controllerMapping.from(Controller.Standard.RightGrip).to(function (value) {
+    gestureLogic(value,1);
+});
+
+function gestureLogic(value,gestureHand){
+    var jointName = "LeftHand";
+    if(gestureHand === 1){
+        jointName = "RightHand";
+    }
+
+    if (value >= 0.7 && isPalmUpwards(gestureHand)) {
+        print("Gesture Start!");
```

Comment:
Remove print statements, or add a PRINT_DEBUG boolean and wrap in there, defaulting to false